### ISSUE TITLE
EIP712Message  warning: Unexpected argument 

### ISSUE THREAD

**Issue by lijiachang** [ApeWorX/eip712#28]
### Environment information

- OS: macOS
- Python Version: 3.8.10
- eip712 : 0.1.5 and 0.1.6

a simple test code:
class Test(EIP712Message):
    _name_: string = "Frank 0x0016"
    _version_: string = "1"
    _chainId_: unit256 = 5
    _verifyingContract_: address = "0x2a980B"

    owner: address
    amount: unit256
    nonce: unit256

mail = Test(owner="0x7f62d", amount=120, nonce=2374)

pycharm warning: <img width="761" alt="Snipaste_2023-01-08_19-58-40" src="https://user-images.githubusercontent.com/28041718/211194839-b5ce2b03-873d-4caf-953b-fe3fcf49b50a.png">

**Comment by lijiachang** [comment]
Although it works, this warning tip is confusing

**Comment by fubuloubu** [comment]
> Although it works, this warning tip is confusing

@lijiachang yes this is currently a problem from using data classy, I am looking to refactor this to a better library this week for a v0.2.0 release, so will check this UX as well

**Comment by fubuloubu** [comment]
solved with #59 if you use class var domain
